src/textual/_region_group.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from collections import defaultdict
+from operator import attrgetter
+from typing import NamedTuple, Iterable
+
+from src.textual.geometry import Region
```

Comment:
Should be a relative import. Can you look in to the editor config that added this.